src/OpenTelemetry.Instrumentation.EventCounters/OpenTelemetry.Instrumentation.EventCounters.csproj

```diff
@@ -7,7 +7,7 @@
         <Nullable>enable</Nullable>
     </PropertyGroup>
     <ItemGroup>
-        <PackageReference Include="OpenTelemetry.Api" Version="1.3.0" />
+        <PackageReference Include="OpenTelemetry.Api" Version="1.3.1" />
```

Comment:
Use `$(OpenTelemetryApiPkgVer)`.

Reply:
we might be able to ship this package as stable, so want to keep reference to the stable one only.

Reply:
That property points to the latest stable version which is `1.3.1` for now. https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/build/Common.props#L35